data/sql/updates/pending_db_world/rev_1689813586760998100.sql

```diff
@@ -0,0 +1,8 @@
+--
+DELETE FROM `spell_script_names` WHERE `ScriptName`='spell_q10651_q10692_book_of_fel_names';
+INSERT INTO `spell_script_names` (`spell_id`,`ScriptName`) VALUES
+(36298, 'spell_q10651_q10692_book_of_fel_names');
+
+DELETE FROM `conditions` WHERE (`SourceTypeOrReferenceId` = 17) AND (`SourceGroup` = 0) AND (`SourceEntry` = 37906) AND (`SourceId` = 0) AND (`ElseGroup` = 0) AND (`ConditionTypeOrReference` = 31) AND (`ConditionTarget` = 1) AND (`ConditionValue1` = 3) AND (`ConditionValue2` = 21178) AND (`ConditionValue3` = 0);
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES
+(17, 0, 37906, 0, 0, 31, 1, 3, 21178, 0, 0, 0, 0, '', 'Rip the pages from this book when Varedis uses Metamorphosis to deprive him of his power.');
```

Comment:
this comment is not descriptive of what the conditions does...

Reply:
I didn't understand about the comment. What is wrong with it?

Reply:
it does not describe the condition. It sounds more like a fairytale

Reply:
It is not a fairy tale. This is the description of the spell, it serves with intention to help understand about it. Although it doesn't sound like the condition, but it talks about the specific target. What is the purpose of it? Is it to draw attention or who is in doubt about targeting

Reply:
change it to
`Book of Fel Names can only target Varedis`

iirc that is what the condition actually does